sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateKeyType.cs

```diff
@@ -29,27 +29,27 @@ public CertificateKeyType(string value)
         }
 
         /// <summary>
-        /// An EC (EllipticCurve) key
+        /// An Elliptic Curve Cryptographic (ECC) algorithm.
```

Comment:
Nice to have: links to more information on these algorithms.

Reply:
I like the idea, and while easier than a similar idea above, the problem here would be the definitive source would be an RFC which aren't the most reader-friendly. We did have links in pre-release documentation to a page on MSDN but it wasn't really helpful and got into technical information about field sizes that, given how keys are created and passed to us, matter little to the majority of developers. I think, once again, a quick search would suffice if developers have questions.